packages/camera/camera/ios/Classes/CameraPlugin.m

```diff
@@ -992,6 +992,18 @@ - (void)handleMethodCallAsync:(FlutterMethodCall *)call result:(FlutterResult)re
     NSDictionary *argsMap = call.arguments;
     NSUInteger cameraId = ((NSNumber *)argsMap[@"cameraId"]).unsignedIntegerValue;
     if ([@"initialize" isEqualToString:call.method]) {
+    int videoFormatValue = ((NSNumber *)argsMap[@"imageFormatGroup"]).unsignedIntegerValue;
+    switch (videoFormatValue) {
+        case 1111970369:
+            videoFormat = kCVPixelFormatType_32BGRA;
+            break;
+        case 875704438:
+            videoFormat = kCVPixelFormatType_420YpCbCr8BiPlanarVideoRange;
+            break;
+        default:
+            videoFormat = kCVPixelFormatType_32BGRA;
+
+    }
```

Comment:
Instead of a number the image format is send as a string so this should be updated to match. Also it would be easier to read and maintain if we can put this in a separate method. Similar to the `getFlashModeForString` method.